translation.json

```
@@ -0,0 +1,111 @@
+{
```

This is the structure it should use, just like the other login components.

```suggestion
{
  widgets: {
    login: {
      fliplet: { }
    }
  }
}
```